src/main/resources/hudson/views/RegExJobFilter/config.jelly

```diff
@@ -3,7 +3,7 @@
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
 	xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
   <f:entry title="${%Regular Expression}:">
-      <f:textbox name="regex" field="regex" checkUrl="'${rootURL}/descriptorByName/hudson.views.RegExJobFilter/checkRegex?value='+encodeURIComponent(this.value)"/>
+    <f:textbox name="regex" field="regex" checkUrl="${rootURL}/descriptorByName/hudson.views.RegExJobFilter/checkRegex" checkDependsOn=""/>

```

Comment:
Thanks for your contribution to this plugin @mawinter69! Unfortunately, there is still one more CSP violation in this plugin after these changes: the `onchange` attribute a few lines down on line 11, which is an inline event handler. Would it be too much to ask for you to fix that up in this PR as well? If that is asking too much, feel free to say so, and in that case I will create a separate Jira ticket to cover that remaining violation.

Reply:
I can look into this in this PR, no problem

Reply:
done. Also fixed the functionality which was broken since 2 years